debian/patches/no-single-process.patch

```diff
@@ -101,18 +101,18 @@ Last-Update: 2024-08-02
  Wants=network-pre.target
  After=hv_kvp_daemon.service
 +After=systemd-remount-fs.service
- Before=auditd.service
```

Comment:
We generally don't want to be modifying systemd service ordering between releases. However, it looks like this just reverts what was recently added, so in this case I think that this is the right thing to do.